templates/sections/_sponsors.html

```diff
@@ -74,7 +74,7 @@ <h3 class="wow fadeInDown">Silver Sponsors</h3>
       <ul class="tg-sponsors col-md-4">
         <li>
           <figure class="tg-sponder">
-             <a data-toggle="modal" href="#zeomega-modal" target="_blank" href="http://www.zeomega.com/">
+             <a href="http://www.zeomega.com/" target="_blank">
```

Comment:
Replace this with the previous code or `<a data-toggle="modal" href="#zeomega-modal">`

Reply:
One task is to link all sponsors through modals as done with ZeOmega

Reply:
I didn't know that. I apologize. As by clicking on zeomega's link it didn't redirect to zeomega's homepage. Which I thought was it's real purpose. I will implement this.

Reply:
Replaced it with previous code